src/index.ts

```diff
@@ -838,6 +838,14 @@ export const right = [
   "tankersley", // Nick Tankersley  - Product Manager
   "vandoren",   // Tim VanDoren     - Sr Software Engineer
   "wilkins",    // Jacob Wilkins    - Principal SecDevOps Engineer
+  "goozy",      // Steph Campagna   - Sr Events Marketing Manager
```

Comment:
@stevelitras the names seem to have been duplicated, see line 841 and then starting again at 845.